### PR TITLE
add filter fields back to policy table

### PR DESCRIPTION
**Pull request by RobDolares**
Related issue: https://github.com/stolostron/backlog/issues/21031
Signed-off-by: rhowingt@redhat.com <rhowingt@redhat.com>
Screen Shot 2022-03-24 at 12 02 56 PM

### CODE REVIEW

File: frontend/src/routes/Governance/policies/Policies.tsx [hunk 1]
```diff
@@ -521,7 +521,7 @@ export default function PoliciesPage() {
     )
 
     const [namespaces] = useRecoilState(namespacesState)
-
+    console.log('tableItems', tableItems)
```
Comment (RobDolares) [inline frontend/src/routes/Governance/policies/Policies.tsx:524]: will remove

**Review by zlayne — DISMISSED** [review]

**Review by fxiang1 — APPROVED** [review]
LGTM!

**Pull request merged**